### PR TITLE
Release speech 1.3.0

### PR DESCRIPTION
**Pull request by tseaver**
This pull request was generated using releasetool.

### CODE REVIEW

File: speech/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,32 @@
...
+
+
+### Implementation Changes
+- Remove gRPC send/recv msg size limit (via synth). ([#8969](https://github.com/googleapis/google-cloud-python/pull/8969))
+
+### New Features
+- Add speaker diarization configuration support to VI (via synth). ([#9202](https://github.com/googleapis/google-cloud-python/pull/9202))
+- Add `SpeakerDiarizationConfig` to V1p1beta1, deprecate `enable_speaker_diarization` and `diarization_speaker_count` (via synth). ([#8795](https://github.com/googleapis/google-cloud-python/pull/8795))
+
+### Dependencies
```
Comment (busunkim96) [inline speech/CHANGELOG.md]:
```suggestion
```

**Comment by busunkim96** [comment]
Please hold on this, it may not be ready for launch...Still looking for the right person to confirm with

**Comment by tseaver** [comment]
@busunkim96 Please merge whenever you get the thumbs-up from the backend team.

**Comment by busunkim96** [comment]
The backend folks do not want speaker diarization published yet. If you would like to use the new feature, please install the library from master.

**Pull request closed without merging**